snapshots/snapshots-processor/src/main/kotlin/com/emergetools/snapshots/processor/PreviewProcessor.kt

```diff
@@ -54,10 +57,13 @@ class PreviewProcessor(
       .functionsWithPreviewAnnotation()
     val multiPreviewAnnotatedFunctions = symbolsWithPreviewAnnotations
```

Comment:
Yeah was about to comment on this, you should be able to remove this now since your `getMultiPreviewAnnotations` will both check current module and submodules.